lib/imapoptions

```diff
@@ -1735,15 +1735,20 @@ If all partitions are over that limit, this feature is not used anymore.
 { "plaintextloginalert", NULL, STRING }
 /* Message to send to client after a successful plaintext login. */
 
-{ "popexpiretime", -1, INT }
-/* The number of days advertised as being the minimum a message may be
+{ "popexpiretime", "-1", DURATION }
```

Comment:
Why isn't this '-1d' rather than '-1'?

Reply:
It seems more natural, when using a negative value as a behaviour flag, to leave off the units? (As in: that's what I would do if I was setting up an imapd.conf for a server.)  And that being the case, I think it makes the docs a bit more clear that the default is NEVER, not a number of days, if the unit is left off the default in this case.

Reply:
Like, the imapd.conf man page ends up looking like this, which I think makes the most sense in this context:

>           popexpiretime: -1
>               The duration advertised as being the minimum a message may be left on the
>               POP  server  before  it  is deleted (via the CAPA command, defined in the
>               POP3 Extension Mechanism, which some clients may support).  This duration
>               [...]